hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/ipc/TestIPC.java

```diff
@@ -881,9 +880,8 @@ private void checkBlocking(int readers, int readerQ, int callQ) throws Exception
     // start server
     final TestServerQueue server =
         new TestServerQueue(clients, readers, callQ, handlers, conf);
-    CallQueueManager<Call> spy = spy(
-        (CallQueueManager<Call>)Whitebox.getInternalState(server, "callQueue"));
-    Whitebox.setInternalState(server, "callQueue", spy);
+    CallQueueManager<Call> spy = spy(server.getCallQueue());
+    server.setCallQueue(spy);
```

Comment:
We can remove `@SuppressWarnings("unchecked")` from this method.

Reply:
I will fix it.